src/core/string_set.cc

```diff
@@ -51,6 +51,38 @@ bool StringSet::Add(string_view src, uint32_t ttl_sec) {
   return true;
 }
 
+void StringSet::AddMany(string_view elems[], size_t count, uint32_t ttl_sec, bool* res) {
```

Comment:
why not `span<string_view>` instead of `elem* and count` ? Also, since `res` is of fixed size, I would also consider returning `bool *` and just have a static variable `bool res[kMaxBatchLen]` 

Again these are preferences so feel free to ignore me :smile: